pmm-app/tests/pages/adminPage.js

```diff
@@ -116,7 +116,7 @@ module.exports = {
     const backgroundColor = await I.grabCssPropertyFrom(element, 'background-color');
 
     assert.strictEqual(
-      backgroundColor[0],
```

Comment:
why it was removed?

Reply:
The new version of codeceptjs added method `grab*FromAll` which is an array. But the `grabCssPropertyFrom` is now taking just one single element. So when you apply `backgroundColor[0]` it is taking one single letter on index 0.
 
![image](https://user-images.githubusercontent.com/63704168/108822896-056afb80-75c0-11eb-80f1-c08fdf8c1c59.png)

Reply:
ok, good